engine/access/rest/middleware/common_query_params.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/gorilla/mux"
 )
 
-const expandQueryParam = "expand"
+const ExpandQueryParam = "expand"
```

Comment:
since this constant is used in the test too

Reply:
yep good idea